types/v1/types.go

```diff
@@ -277,7 +277,17 @@ type VolumeAPISpec struct {
 	} `yaml:"metadata"`
 }
 
-// -------------Snapshot Structs ----------
+// SnapshotAPISpec hsolds the config for creating asnapshot of volume
+type SnapshotAPISpec struct {
+	Kind       string `yaml:"kind"`
+	APIVersion string `yaml:"apiVersion"`
+	Metadata   struct {
```

Comment:
Let the Kind, APIVersion & Metadata follow the rules followed by Volume.

What is the reason for a change for snapshot struct ?